packages/react/src/components/Chip/Chips.tsx

```diff
@@ -62,16 +61,12 @@ export const ChipRemovable = forwardRef<HTMLButtonElement, ChipRemovableProps>(
  * <Chip.Checkbox name="language" value="bokmål">Bokmål</Chip.Checkbox>
  */
 export const ChipCheckbox = forwardRef<HTMLLabelElement, ChipCheckboxProps>(
-  function ChipCheckbox({ asChild, children, className, size, ...rest }, ref) {
+  function ChipCheckbox({ asChild, children, className, ...rest }, ref) {
     const inputType = (rest as { type?: string }).type ?? 'checkbox';
     const Component = asChild ? Slot : 'label';
 
     return (
-      <Component
-        className={cl('ds-chip', className)}
-        data-size={size}
-        ref={ref}
-      >
+      <Component className={cl('ds-chip', className)} ref={ref}>
```

Comment:
Missing `data-size` on `Component`. Now ends up at the wrong level, not scaling default `label` element

Reply:
hm, `...rest` is being sent to the input element 🤔